src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIIACMC.scala

```diff
@@ -410,19 +496,21 @@ object EXUIIACMC {
 
   val shareacase =
 
-    group("XUI${service}_260_ShareACase") {
-      exec(http("XUI${service}_260_005_ShareACase")
+    group("XUI${service}_270_ShareACase") {
+      exec(http("XUI${service}_270_005_ShareACase")
         .get("/api/caseshare/cases?case_ids=${caseId}")
-        .headers(IACHeader.headers_scase1)
+        .headers(Headers.commonHeader)
+        .header("accept", "application/json, text/plain, */*")
         .check(status.in(200, 304))
         .check(jsonPath("$..email").find(0).optional.saveAs("user0"))
         .check(jsonPath("$..firstName").find(0).optional.saveAs("firstName"))
         .check(jsonPath("$..lastName").find(0).optional.saveAs("lastName"))
         .check(jsonPath("$..idamId").find(0).optional.saveAs("idamId")))
     
-      .exec(http("XUI${service}_260_010_ShareACaseUsers")
+      .exec(http("XUI${service}_270_010_ShareACaseUsers")
         .get("/api/caseshare/users")
-        .headers(IACHeader.headers_scase1)
+        .headers(Headers.commonHeader)
+        .header("accept", "application/json, text/plain, */*")
         .check(status.in(200, 304))
         .check(jsonPath("$..email").find(0).optional.saveAs("user1"))
```

Comment:
Either these checks should be made not optional, or there should be logic to not call the Share A Case functionality below if they're not found. The Share A Case JSON file demands these values.